static/app/components/carousel.tsx

```diff
@@ -94,21 +94,16 @@ function Carousel({children}: Props) {
 
 const CarouselContainer = styled('div')`
   position: relative;
-  padding-bottom: ${space(0.5)};
 `;
 
 const CarouselItems = styled('div')`
   display: flex;
   overflow-x: scroll;
   scroll-behavior: smooth;
-  &::-webkit-scrollbar {
-    background-color: transparent;
-    height: 8px;
-  }
-  &::-webkit-scrollbar-thumb {
-    background: ${p => p.theme.gray300};
-    border-radius: 8px;
-  }
+  /* We provide some margin to make room for the scroll bar. It is applied on
+   * the top and bottom for consistency.
+   */
```

Comment:
does the scrollbar look okay on windows if we don't explicitly style it?

Reply:
@vuluongj20 any feelings about this?

Reply:
Generally I don't think we should restyle the scrollbar unless there's an explicit need to do so. Reverting to default browser styles would just make the experience more consistent and expected for Windows and Mac users alike.